public/i18n/es-es.json

```diff
@@ -13,7 +13,7 @@
   "ftm-accepts-open-boxes" : "Se acepta cajas / bolsas abiertas",
   "ftm-ack-data-sources" : "Este sitio también incluye datos de <a href=\"https://tinyurl.com/hospitals-needing-ppe\"> https://tinyurl.com/hospitals-needing-ppe </a> , un proyecto de:",
   "ftm-add-donation-site-answer" : "<a class='add-donation-site-form'> <button style='margin-top:20px;' type='button' class='btn-sm btn-primary'> Complete este formulario </button> </a>",
-  "ftm-add-donation-site-header" : "¿Necesita EPP ( equipo de protección personal)?",
+  "ftm-add-donation-site-header" : "¿Necesita EPP (equipo de protección personal)?",
```

Comment:
@ywyw did you add this in yourself or did this come in from Smartling? We should only be updating en.json and then our translation platform will update other languages. If there's an issue with es / de can you post in #languages so that the translators can update?